src/layouts/sidebar-sidecar/server.ts

```diff
@@ -218,17 +237,39 @@ export function getStaticGenerationFunctions<
           menuItems: navDataWithFullPaths,
           // TODO: won't default after `BASE_PATHS_TO_NAMES` is replaced
           title: BASE_PATHS_TO_NAMES[basePath] || product.name,
-          overviewItemHref: `/${product.slug}/${basePath}`,
+          overviewItemHref: versionPathPart
+            ? `/${product.slug}/${basePath}/${versionPathPart}`
+            : `/${product.slug}/${basePath}`,
         },
       ]
 
+      /**
+       * Generate the arguments sent to `getDocsBreadcrumbs` based on whether or
+       * not there is a version in the current path.
+       */
+      let generatedBaseName
+      let generatedBasePath
+      let filteredPathParts
+      if (indexOfVersionPathPart >= 0) {
+        generatedBaseName = `${baseName} ${versionPathPart}`
```

Comment:
Yes, I included it with the root path breadcrumb item because it doesn't seem like it's needing to be its own item. It also wasn't accurate to make it the last breadcrumb item, since it's not the last item in our URL paths for versioned pages. What are your thoughts on this?

Reply:
Makes sense! I feel like this could be something that comes up in QA or feedback based on the designs, should we propose this adjustment before we go ahead with it?

Reply:
Got the OK via [Slack](https://hashicorp.slack.com/archives/C01KCU4HDPY/p1654118539129629?thread_ts=1654117966.138619&cid=C01KCU4HDPY)!